Client/sdk/game/CWorld.h

```diff
@@ -51,6 +51,14 @@ struct SLineOfSightBuildingResult
     CEntitySAInterface* pInterface;
 };
 
+struct SProcessLineOfSightMaterialInfoResult {
+    CVector2D   uv;            // On-texture UV coordinates of the intersection point
+    const char* textureName;   // GTA texture name
+    const char* frameName;     // The name of the frame the hit geometry belongs to
```

Comment:
`textureName` and `frameName` should be initialized I guess.

Reply:
```cpp
bool valid{}; // Data found in this struct is only valid if this is `true`!
```